### DISCUSSION TITLE
zepeto 스튜디오 내보내기 문제

### DISCUSSION THREAD

**Discussion by Lumis015** [naverz/zepeto-studio-kor#379] [5. Q&A]
누구든지이 문제가 발생 했습니까? 아니면 저만 그런가요?
이제 Unity 2020.3.9 및 FileFolder 3.1.11.1(2022.01.27 업데이트)을 사용합니다.
FBX를 ZPT 형식 파일로 내보내는 것은 5MB 단위로 증가하므로 스토어(최대 1MB)에 업로드할 수 없기 때문에 혼란스럽습니다.

TT 답변 부탁드립니다. 대단히 감사합니다.

(저는 구글 번역기를 사용하기 때문에 조금 이상할 수 있습니다)

**Comment by Zepeto-TechEvangelist** [comment]
#322 It's the same as the issue.
please check if the problem still occurs in the latest version.
